nixos/modules/services/backup/syncoid.nix

```diff
@@ -123,9 +123,7 @@ in
     };
 
     sshKey = mkOption {
-      type = types.nullOr types.path;
-      # Prevent key from being copied to store
-      apply = mapNullable toString;
+      type = types.nullOr types.str;
```

Comment:
Now that I think about it, users really shouldn't be passing `./some-file` style paths here unless they are sure they know what they're doing, so perhaps we should break this (with a changelog entry).

If they do, it will only work if the file containing the SSH key file is located somewhere readable by the syncoid user, and that also assumes they aren't doing deploying the config remotely and they don't move or delete the file later. Additionally, it they are using flakes, the file will still be copied to the store, despite what the comment says.